api/internal/target/kusttarget_configplugin.go

```diff
@@ -48,13 +48,13 @@ func (kt *KustTarget) configureBuiltinTransformers(
 	tc *builtinconfig.TransformerConfig) (
 	result []resmap.Transformer, err error) {
 	for _, bpt := range []builtinhelpers.BuiltinPluginType{
+		builtinhelpers.PatchJson6902Transformer,
 		builtinhelpers.PatchStrategicMergeTransformer,
 		builtinhelpers.PatchTransformer,
 		builtinhelpers.NamespaceTransformer,
```

Comment:
While I agree that the `patchJson6902Transformer` should be placed next to the other patch transformers for consistency in behavior, I'm wondering if there might be any other unintended or possibly undesirable consequences for changing the order?